flashinfer/trace_apply/__init__.py

```
@@ -0,0 +1,33 @@
+    register_plan_run,
+)
+from .config import ApplyConfig
+
+__all__ = [
+    "ApplyError",
+    "ApplyConfig",
+    "enable_apply",
+    "enable_apply_from_env",
+    "disable_apply",
+    "register_plan_run",
+]
```

Ruff (0.15.12)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)